sbag/generators/java/templates/__project__/src/main/resources/application.properties.jinja

```diff
@@ -0,0 +1,8 @@
+spring.datasource.driverClassName=org.postgresql.Driver
+spring.datasource.url= jdbc:postgresql://localhost:5432/test_db
+spring.datasource.initialization-mode=always
+spring.datasource.platform=postgres
+spring.datasource.username=postgres
+spring.datasource.password=ftn
```

Comment:
Should we set password to be enviroment variable?

Reply:
I agree